packages/config/config/devices/0x0086/templates/aeotec_template.json

```diff
@@ -1567,6 +1588,760 @@
+		"$import": "#base_led_duration",
+		"label": "Light Effect No. 6: LED Indicator Off Duration",
+		"defaultValue": 0
+	},
+	"led_duration_on_7": {
+		"$import": "#base_led_duration",
+		"label": "Light Effect No. 7: LED Indicator On Duration",
+		"defaultValue": 0
+	},
+	"led_duration_off_7": {
+		"$import": "#base_led_duration",
+		"label": "Light Effect No. 7: LED Indicator Off Duration",
+		"defaultValue": 1
+	},
+	"base_basic_set_command": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 4,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "Start playing -> On; Stop playing -> None",
+				"value": 1
+			},
+			{
+				"label": "Start playing -> Off; Stop playing -> None",
+				"value": 2
+			},
+			{
+				"label": "Start playing -> On; Stop playing -> Off",
+				"value": 3
+			},
+			{
+				"label": "Start playing -> Off; Stop playing -> On",
+				"value": 4
+			}
+		]
+	},
+	"basic_set_command_browse": {
+		"$import": "#base_basic_set_command",
+		"label": "Group 2 Basic Set Command (Browse)",
+		"defaultValue": 3
+	},
+	"basic_set_command_tamper": {
+		"$import": "#base_basic_set_command",
+		"label": "Group 3 Basic Set Command (Tampering)",
+		"defaultValue": 3
+	},
+	"basic_set_command_doorbell_1": {
+		"$import": "#base_basic_set_command",
+		"label": "Group 4 Basic Set Command (Doorbell 1)",
+		"defaultValue": 3
+	},
+	"basic_set_command_doorbell_2": {
+		"$import": "#base_basic_set_command",
+		"label": "Group 5 Basic Set Command (Doorbell 2)",
+		"defaultValue": 3
+	},
+	"basic_set_command_doorbell_3": {
+		"$import": "#base_basic_set_command",
+		"label": "Group 6 Basic Set Command (Doorbell 3)",
+		"defaultValue": 3
+	},
+	"basic_set_command_environment": {
+		"$import": "#base_basic_set_command",
+		"label": "Group 7 Basic Set Command (Environment)",
+		"defaultValue": 3
+	},
+	"basic_set_command_security": {
+		"$import": "#base_basic_set_command",
+		"label": "Group 8 Basic Set Command (Security)",
+		"defaultValue": 3
+	},
+	"basic_set_command_emergency": {
+		"$import": "#base_basic_set_command",
+		"label": "Group 9 Basic Set Command (Emergency)",
+		"defaultValue": 3
+	},
+	"button_unpair": {
+		"label": "Button Unpairing",
+		"valueSize": 1,
+		"minValue": 1,
+		"maxValue": 7,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Unpair Button No. 1",
+				"value": 1
+			},
+			{
+				"label": "Unpair Button No. 2",
+				"value": 2
+			},
+			{
+				"label": "Unpair Button Nos. 1-2",
+				"value": 3
+			},
+			{
+				"label": "Unpair Button No. 3",
+				"value": 4
+			},
+			{
+				"label": "Unpair Button Nos. 1 & 3",
+				"value": 5
+			},
+			{
+				"label": "Unpair Button Nos. 2 & 3",
+				"value": 6
+			},
+			{
+				"label": "Unpair Button Nos. 1-3",
+				"value": 7
+			}
+		]
+	},
```

Comment:
Does it make sense to turn this into partials?
Unpair 1 = 1
Unpair 2 = 2
Unpaif 2 = 4

Reply:
If not, does it make sense to add an option for value 0 like in the one below?

Reply:
I changed it like the one below. There are already so many parameters I hate to make six more for something not used that frequently...this way they can select to enter pairing or unpairing on a specific button.